plasmapy/formulary/dimensionless.py

```diff
@@ -8,7 +8,12 @@
 highly turbulent, while turbulence is negligible at low Reynolds
 numbers.
 """
-__all__ = ["beta", "quantum_theta", "Reynolds_number", "Mag_Reynolds"]
+__all__ = [
+    "beta",
+    "quantum_theta",
+    "Reynolds_number", "Re",
+    "Mag_Reynolds", "Rm",
```

Comment:
I see where you're going with this - keeping the aliases on the same line is neat. However, our `black` code formatter is complaining (as it probably should be, this is not exactly a standard usage)!.

Subjectively: I think given that it'll still be ordered in this way in the docs, it's fine to keep them on multiple lines - just `pip install black` and run it on the `plasmapy` directory.

@rocco8773 @namurphy, do you have any opinions on this?

Reply:
@StanczakDominik I think I'm as conflicted on this as you are.  Grouping is nice and it links the association, but it's also non-standard.  Personally, I'd probably stick to an alphabetized list and then add to the docstring of the parent function (e.g. `Reynolds_number`) an alias comment (e.g. `Re`).  I think an un-alphabetized table/list in the documentation is always confusing to new users, and returning users.

I may have just talked myself out of being conflicted. 😆

Reply:
I like this solution! It is a bit of work, but that's better than having the functions be completely invisible to each other except for the main documentation page for the formulary and other modules.

Reply:
Added aliases in function docstrings in f1aa1d4